npep/npep-126-egress-traffic-control.md

```diff
@@ -0,0 +1,90 @@
+# NPEP-126: Add northbound traffic support in (B)ANP API
+
+* Issue: [#126](https://github.com/kubernetes-sigs/network-policy-api/issues/126)
+* Status: Provisional
+
+## TLDR
+
+This NPEP proposes adding support for cluster egress (northbound) traffic control
+in the `AdminNetworkPolicy` and `BaselineAdminNetworkPolicy` API objects.
+
+## Goals
+
+* Implement egress traffic control towards external destinations (outside the cluster)
+  - NOTE: Currently there are no user stories for `Pass` action with northbound traffic
+    but users will get this for free since this action is anyways supported in the API.
```

Comment:
(I don't think the thing about Pass needs to be explicitly stated like that.)

Reply:
yeah I debated putting this here versus not putting it, the only reason why I put it is to document this somewhere that this was thought of and in the end we decided to implement pass anyways. but i can remove this note from the npep and put it on the PR commit message instead since that will also maintain the history of discussion without spoiling the NPEP.

Reply:
this bit has been removed and added into the PR message instead.